### PR TITLE
Change TX V3 Deregister To Bool

### PR DESCRIPTION
**Pull request by Doy-lee**
@tewinget and @jcktm

### CODE REVIEW

File: src/cryptonote_core/blockchain.cpp [hunk 1]
```diff
@@ -2693,7 +2693,10 @@ bool Blockchain::check_tx_inputs(transaction& tx, tx_verification_context &tvc,
     }
 
     // min/max tx version based on HF, and we accept v1 txes if having a non mixable
-    const size_t max_tx_version = (hf_version <= 3) ? 1 : 2;
+    const size_t max_tx_version = (hf_version <= 3) ? 1 : (hf_version < 9)
```
Comment (tewinget) [inline src/cryptonote_core/blockchain.cpp]: While I'm not a fan of nested ternary statements, I suppose this is fine.

File: src/serialization/json_object.cpp [hunk 2]
```diff
@@ -219,6 +219,8 @@ void toJsonValue(rapidjson::Document& doc, const cryptonote::transaction& tx, ra
 
   INSERT_INTO_JSON_OBJECT(val, doc, version, tx.version);
   INSERT_INTO_JSON_OBJECT(val, doc, unlock_time, tx.unlock_time);
+  INSERT_INTO_JSON_OBJECT(val, doc, output_unlock_times, tx.output_unlock_times);
```
Comment (tewinget) [inline src/serialization/json_object.cpp]: Oh, I forgot this; shame on me, and thanks!

**Comment by tewinget** [comment]
I prefer to avoid PRs with mixed purpose/context, but in this case it'll be fine.

**Comment by jcktm** [comment]
loki-per-output-unlock needs rebasing

**Pull request merged**